packages/usql/build.sh

```diff
@@ -0,0 +1,35 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/xo/usql
+TERMUX_PKG_DESCRIPTION="A universal command-line interface for SQL databases"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@flipee"
+TERMUX_PKG_VERSION="0.18.1"
+TERMUX_PKG_SRCURL=https://github.com/xo/usql/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=692a3b30027b9a5bf8c814fbb051a7cf62f1b4895c397ae7bcc3a030cafd6f31
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make() {
+	termux_setup_golang
+	export GOPATH=$TERMUX_PKG_BUILDDIR
+
+	cd $TERMUX_PKG_SRCDIR
+
+	$TAGS="most no_adodb no_duckdb"
```

Comment:
There shouldn't be a `$` in front of `TAGS` here since its an assignment, this is what caused the build to fail

Reply:
Yeah, I know hehe. I'm fixing the build errors that appeared for other ARCHs (I'm on my phone 😅).

Reply:
No worries